stripe_test.go

```diff
@@ -156,9 +156,10 @@ func TestUserAgent(t *testing.T) {
 
 func TestUserAgentWithAppInfo(t *testing.T) {
 	appInfo := &stripe.AppInfo{
-		Name:    "MyAwesomePlugin",
-		URL:     "https://myawesomeplugin.info",
-		Version: "1.2.34",
+		Name:      "MyAwesomePlugin",
+		PartnerId: "partner_1234",
```

Comment:
since you changed the input here, you should update the test below though to ensure it's passed properly

Reply:
@remi-stripe The confusion here is partly my mistake — Zach had originally added the new partner ID to `User-Agent`, but I asked him to take it out again because it didn't really seem to fit the format of that field very well. Therefore, the new field is _only_ sent as a new field in the specially encoded `X-Stripe-User-Agent` header instead.

The test below only checks `User-Agent`, which is why it wasn't updated I think.

Reply:
Oh I see. Should we add a test for it to be safe since it's not really an API parameter?

Reply:
Yeah I guess it can't hurt. I added #609 to bring in some basic tests for the existing parameters. We can merge that first and then have Zach rebase and add one new assertion for "partner ID".